helpers/integration.go

```diff
@@ -265,20 +304,20 @@ func GetIntegrationPipelineRunOutcome(adapterClient client.Client, ctx context.C
 	return &IntegrationPipelineRunOutcome{
 		pipelineRunSucceeded: true,
 		pipelineRun:          pipelineRun,
-		results:              map[string]*AppStudioTestResult{},
+		results:              map[string]*IntegrationTestTaskResult{},
 	}, nil
 }
 
 // GetAppStudioTestResultsFromPipelineRunWithChildReferences finds all TaskRuns from childReferences of the PipelineRun
 // that also contain a TEST_OUTPUT result and returns the parsed data
 // returns map taskName: result
-func GetAppStudioTestResultsFromPipelineRunWithChildReferences(adapterClient client.Client, ctx context.Context, pipelineRun *tektonv1.PipelineRun) (map[string]*AppStudioTestResult, error) {
+func GetAppStudioTestResultsFromPipelineRunWithChildReferences(adapterClient client.Client, ctx context.Context, pipelineRun *tektonv1.PipelineRun) (map[string]*IntegrationTestTaskResult, error) {
```

Comment:
Should we change the name from GetAppstudioTestResults to something like GetIntegrationTestTaskResult? Since the return value changed?

Reply:
updated